### PR TITLE
Add requirements page of get-started section for kor doc

### PR DESCRIPTION
**Pull request by songkg7**
## Summary

Translate `Requirements` documentation into Korean related to #864 

## How Has This Been Tested?

execute `npm start`

## Description

For the title, I didn't translate it, and I was wondering if there is a rule for that.

### CODE REVIEW

**Comment by SooKim1110** [comment]
기여 감사드립니다! 🚀

File: docs/content/v1.0.x-kor/docs/get-started/requirements.md [hunk 1]
```diff
@@ -8,4 +8,56 @@ docs:
 weight: 21
 ---
 
-### 한국어 번역 필요
+{{< alert icon="💡" text="Fixture Monkey 는 테스트 환경용으로 설계되었습니다. 운영 코드에는 포함되지 않도록 하는게 좋습니다." />}}
```
Comment (SooKim1110) [inline docs/content/v1.0.x-kor/docs/get-started/requirements.md:11]: `운영 코드에는 포함하지 않을 것을 권장합니다.` 가 조금 더 명확할 것 같습니다!

File: docs/content/v1.0.x-kor/docs/get-started/requirements.md [hunk 2]
```diff
@@ -8,4 +8,56 @@ docs:
...
+## 종속성
+
+| Dependency                    | Description                                  |
+|-------------------------------|----------------------------------------------|
+| fixture-monkey                | Core library                                 |
+| fixture-monkey-starter        | Starter dependency for fixture monkey        |
+| fixture-monkey-kotlin         | Kotlin support                               |
+| fixture-monkey-starter-kotlin | Starter dependency for fixture monkey kotlin |
+
+**fixture-monkey-starter** 는 Fixture Monkey 를 시작하는 데 도움이 되도록 fixture-monkey-jakarta-validation과 같은 플러그인들과 함께 제공되는 스타터 패키지입니다.
```
Comment (SooKim1110) [inline docs/content/v1.0.x-kor/docs/get-started/requirements.md:30]: `플러그인들이` 로 수정해주시면 더 좋을 것 같습니다!

File: docs/content/v1.0.x-kor/docs/get-started/requirements.md (hunk 1 above)
Comment (songkg7) [reply docs/content/v1.0.x-kor/docs/get-started/requirements.md]: 그렇네요. 피드백 감사합니다! 👍

**Review by SooKim1110 — APPROVED** [review]
👍

**Pull request merged**